otx/cli/tools/explain.py

```diff
@@ -65,18 +68,61 @@ def get_args():
         "For Openvino task, default method will be selected.",
     )
     parser.add_argument(
-        # "-w",
+        "--process-saliency-maps",
```

Comment:
Can you please add these parameters to the documentation ([CLI commands: Explain](https://github.com/openvinotoolkit/training_extensions/blob/develop/docs/source/guide/get_started/quick_start_guide/cli_commands.rst#explanation)) to keep it up to date?

Also can you consider updating [Changelog](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md#v120) with you CLI update as well?